specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -0,0 +1,1045 @@
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ResourceNameInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"name": "createQuotaRequest",
+						"in": "body",
+						"description": "Quota requests payload.",
+						"required": true,
+						"schema": {
+							"$ref": "#/definitions/CurrentQuotaLimit"
+						}
+					},
+					{
+						"$ref": "#/parameters/IfMatchRequiredParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Returns Quota requests details.",
+						"schema": {
+							"$ref": "#/definitions/QuotaRequestDetails"
+						}
+					},
+					"202": {
+						"description": "The request is accepted and is being processed. Please use requestId to check the status.",
+						"schema": {
+							"$ref": "#/definitions/QuotaRequestSubmitResponse"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Capacity/providers/{providerId}/locations/{location}/serviceLimits": {
+			"get": {
+				"tags": [
+					"QuotaInformation"
+				],
+				"summary": "Gets the current quota limit and usages for all the resources by the resource provider at the specified location.",
+				"description": "This API gets the current quota limits and usages for the resource provider for the specified location. This response can be used to submit quotaRequests.",
+				"operationId": "Quotas_listStatus",
+				"x-ms-examples": {
+					"Quotas_listUsagesForCompute": {
+						"$ref": "./examples/getComputeUsages.json"
+					},
+					"Quotas_listUsagesForBatchAI": {
+						"$ref": "./examples/getBatchAIUsages.json"
+					},
+					"Quotas_listUsagesForMsSql": {
+						"$ref": "./examples/getMsSqlUsages.json"
+					}
+				},
+				"parameters": [
+					{
+						"$ref": "#/parameters/SubscriptionIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ProviderIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK. Quota information will be returned as response with pagination.",
+						"schema": {
+							"$ref": "#/definitions/QuotaLimits"
+						},
+						"headers": {
+							"ETag": {
+								"description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+								"type": "string"
+							}
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				},
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			},
+			"put": {
```

Comment:
what is this PUT? the API signature is not correct. You have a PUT on serviceLImits/{name} what is the intent of this PUT?

Reply:
This to put quota request for one resource.
Example- Put - …/Microsoft.Cpmpute/eastus/standardNDSFamily

 {
				"limit": 20,
				"unit": "Count",
				"currentValue":10,
				
This will be used by BluePrint, as you suggested earlier.

Reply:
Does it sound fine?